libyang/data.py

```diff
@@ -204,6 +204,7 @@ def __init__(self, context: "libyang.Context", cdata):
         """
         self.context = context
         self.cdata = cdata  # C type: "struct lyd_node *"
+        self.cdata_attrs = []  # C tupe: "struct lyd_attr *"
```

Comment:
Please fix the typo here ("tupe" => "type").

Reply:
Done in the DNodeAttrs class